app/src/main/res/layout/contract_detail_activity.xml

```diff
@@ -50,6 +51,16 @@
                         android:id="@+id/cardContainer"
                         layout="@layout/insurance_contract_card" />
 
+                    <TextView
+                        android:id="@+id/termination_info"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:padding="@dimen/base_margin_double"
+                        android:text="@string/INSURANCE_DETAILS_TERMINATION_INFO"
+                        android:textSize="13sp"
```

Comment:
We should use a `textAppearance` here I think, instead of inlining a `textSize` - looks like the specified text style does not exist in our design system, we should probably check that with Hiro 😄